runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettingsInternal.java

```diff
@@ -95,6 +106,8 @@
     private boolean mSpatialNavigationEnabled = true;
     private boolean mQuirksModeEnabled = false;
 
+    private LayoutAlgorithmInternal mLayoutAlgorithm = LayoutAlgorithmInternal.NARROW_COLUMNS;
```

Comment:
Yes, from [WebSettings.LayoutAlgorithm](http://developer.android.com/reference/android/webkit/WebSettings.LayoutAlgorithm.html), `NORMAL` is recommended.
From [AwSettings.java#L55](https://github.com/crosswalk-project/chromium-crosswalk/blob/master/android_webview/java/src/org/chromium/android_webview/AwSettings.java#L55), the default value is still `NARROW_COLUMNS`.
Thanks.